ztp/policygenerator/utils/filesHandler.go

```diff
@@ -51,6 +52,11 @@ func (fHandler *FilesHandler) ReadSourceFile(fileName string) ([]byte, error) {
 	return fHandler.readFile(fHandler.sourceDir + "/" + fileName)
 }
 
+// Optional override of the base directory containing the resources directory. Used for testing.
+func (fHandler *FilesHandler) SetResourceBaseDir(path string) {
```

Comment:
Would you instead create a acmPolicyTemplate.go file and set resource/acm-policy-template.yaml content as constant string. what do u think ?

Reply:
Not sure I understand. This function allows me to set the path where resource files are pulled from in case they aren't found in the default location. It is used just for testing purposes but is more flexible than the  Getwd implementation put in previously (easier to use this way when the tests are in a sub-directory as in policyGen).